includes/AssignPostTable.php

```diff
@@ -131,7 +135,7 @@
 		else
 		{
 			$stmt2 = $sqlConn->prepare("UPDATE $assignPostTable SET $assignedUser = ? WHERE $postTitle = \"$row[0]\"");
-			$stmt2->bind_param ('s', $_POST[$row[0]] );
+			$stmt2->bind_param ('s', $_POST[$newRow] );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->